asllib/Parser.mly

```diff
@@ -372,33 +376,28 @@ let int_constraint ==
   | ~=expr;                     < Constraint_Exact >
   | e1=expr; SLICING; e2=expr;  < Constraint_Range >
 
-let expr_pattern := make_expr (expr_pattern)
-
-let pattern_set ==
-  | annotated (
-      BNOT; ~=braced(pattern_list); < Pattern_Not >
-    )
-  | braced(pattern_list)
 
+let expr_pattern := make_expr (expr_pattern)
+let pattern_mask == annotated(~=MASK_LIT; < Pattern_Mask >)
 let pattern_list == annotated(~=nclist(pattern); < Pattern_Any >)
+
 let pattern :=
   annotated (
     | ~=expr_pattern; < Pattern_Single >
     | e1=expr_pattern; SLICING; e2=expr; < Pattern_Range >
     | MINUS; { Pattern_All }
     | LEQ; ~=expr; < Pattern_Leq >
     | GEQ; ~=expr; < Pattern_Geq >
-    | ~=MASK_LIT; < Pattern_Mask >
     | ~=plist2(pattern); < Pattern_Tuple >
   )
+  | pattern_mask
```

Comment:
Just curious: why did you move `pattern_mask` outside of this `annotated` block and into its own. It seems equivalent, no?